### ISSUE TITLE
Juliaup doesn't create config file automatically

### ISSUE THREAD

**Issue by simeonschaub** [JuliaLang/juliaup#236]
I downloaded the tarballs for 1.5.31 and extracted it to `/root/.local/bin`. I then get the following error when I start `julialauncher`:

```
root@localhost:~# julialauncher
Error: The Julia launcher failed to load a configuration file.

Caused by:
    Could not open self configuration file "/root/.local/juliaupself.json": Os { code: 2, kind: NotFound, message: "No such file or directory" }
```

I thought Juliaup should create that file automatically if it doesn't exist, no?

**Comment by davidanthoff** [comment]
I think the root problem is that we need to compile one version for download from the github release page that doesn't include the selfupdate feature. Versions that include `selfupdate` really need to be installed via `juliainstaller`, so we shouldn't have those versions on the github release page.

**Comment by cailinw** [comment]
Hello, thanks for working on this issue! I'm running into this as well after running `juliaup self update`, and the error now occurs every time I run `julialauncher`.

What would be the temporary fix for this? Should we re-install juliaup and then not use the `self update` command until this is resolved?

Thank you.

**Comment by davidanthoff** [comment]
@cailinw so just to be clear: if you install via `curl -fsSL https://install.julialang.org | sh`, then `self update` should just work, and I would recommend that (or `brew install juliaup` if you are on Mac).

If you download `juliaup` from the github release page here, just don't use `self update`, that will never work. I just need to make sure that the version of the files that we host on github don't even support that command.

**Comment by Firionus** [comment]
I had the same problem as @cailinw. I had an old version of juliaup on Ubuntu 21.10. I installed it some time ago when @davidanthoff released the first easy install version for Linux in https://github.com/JuliaLang/juliaup/issues/84#issuecomment-975185070 (worked like a charm btw). 

Today, I ran `juliaup self update` and got the issue with the missing config file. Unfortunately, I'm not sure how to install that old version again to reproduce and the current installer only seems to install the current version, so `juliaup self update` does nothing.

Anyway, I removed `~/.juliaup` and re-installed with `curl -fsSL https://install.julialang.org | sh`. Works fine again.
